### ISSUE TITLE
Won't Connect to BakaBT via Jackett Using Torznab Custom.

### ISSUE THREAD

**Issue by SCRPD** [Radarr/Radarr#1430]
**Description:**
I use BakaBT & Jackett on Sonarr & it works fine. With Radarr it's having an issue. I posted both version & part of the log file below. 
Thanks.

**Radarr Version:**
Radarr Ver. 0.2.0.654
Jackett Version 0.7.1308.0

**Logs:**
[v0.2.0.654] System.InvalidOperationException: Sequence contains no elements
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.TestConnection() in C:\projects\radarr-usby1\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 324

17-4-22 17:07:07.1|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- Unable to connect to indexer, check the log for more details
17-4-22 17:07:08.5|Warn|Torznab|Unable to connect to indexer

**Comment by Quiksmage** [comment]
I'm getting the same error as above attempting to add the jackett torznab.

I did notice that in jackett, 5070 - TV/Anime is listed as its category.

I attempted to add 5070 to the category line when adding the indexer, but it had no change.

Is it likely that the 5070 category is causing this problem considering it isn't movie related?

If that is the case, the Jackett maintainer would have to add 2000 - Movies, to the bakabt entry since we can't edit this. 

Animebytes, for instance, has 2000 - Movies, and I was successfully able to add animebytes to Radarr via Jackett Torznab.
